### ISSUE TITLE
Icon Conversion: Custom Icons to Standard Icons

### ISSUE THREAD

**Issue by otaree** [activist-org/activist#303]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue focuses on optimizing icon usage within the project by transitioning from custom icon components to standard icons. The aim is to enhance consistency and maintenance. The suggestion involves incorporating standard icons like <Icon name='bi:camera-video' /> or Bootstrap Icons for non-branded situations.

### Contribution

Happy to support someone on this issue or work on it myself

**Comment by andrewtavis** [comment]
Thank you, @otaree! Let's finish up #299 and then if you'd like to start on this one we can look into where the changes need to happen 😊

**Comment by andrewtavis** [comment]
Just assigned you given our discussion in #299, @otaree!

**Comment by andrewtavis** [comment]
Looking at the current custom icons, @otaree, I think we have viable options for a few of them:

- `IconDiscussion`: https://icones.js.org/collection/all?s=chat (`ph:chats-light`)
- `IconLegal`: https://icones.js.org/collection/all?s=scale (`cil:balance-scale`)

The other ones I didn't find very good options for, or they're branded :) Note further that we'd need to make sure that the licenses for them are included in [NOTICE.txt](https://github.com/activist-org/activist/blob/main/NOTICE.txt). I can do this and also check to make sure that we have all icons included in there so far 😊

**Comment by otaree** [comment]
I've found some possible replacements for our custom icons:

1. IconOrganization: https://icones.js.org/collection/all?s=group
2. IconFAQ: https://icones.js.org/collection/all?s=question
3. IconResource: https://icones.js.org/collection/all?s=wrench

Please let me know your thoughts on these icon choices. If they align well with our design and messaging, we could potentially use them to replace our current custom icons. Looking forward to your feedback!

**Comment by andrewtavis** [comment]
I was looking for some of those as well, @otaree :) Thanks for checking for them too! Thing is that for each of them there's not really a good option that fully replaces what the current one does. For organizations there's none that has people in a rounded square; for faqs none with a question in a talk bubble; and for resources there's none that also includes a pencil, as some resources would be written (especially at the start). I think we're fine with the current shifts in this 😊

I'll get to the review shortly!

**Comment by andrewtavis** [comment]
The FAQ one that we have is a combination of some things from our main one, bootstrap. I'm seeing now that there are a couple that are in chat bubbles, but they're just not quite right as far as not having rounded corners for the question mark dot or just looking a bit weird 🤔

**Comment by andrewtavis** [comment]
218d3a8 finalizes this, @otaree :) I actually reverted back to the legal icon as I just didn't like how the others were looking. Plus when I looked at the designs I remembered that I actually made this icon myself 😇 I also expanded out the [NOTICE.txt](https://github.com/activist-org/activist/blob/main/NOTICE.txt) file with all of the needed licenses from the icon sets, so it's great that we got to this! Thank you!
